apollo-router-core/src/lib.rs

```diff
@@ -26,31 +26,37 @@ mod cache;
 mod error;
 mod json_ext;
 mod naive_introspection;
-mod query;
 mod query_cache;
 mod query_planner;
 mod request;
 mod response;
-mod schema;
+mod spec;
 mod traits;
 
 pub use cache::*;
 pub use error::*;
 pub use json_ext::*;
 pub use naive_introspection::*;
-pub use query::*;
 pub use query_cache::*;
 pub use query_planner::*;
 pub use request::*;
 pub use response::*;
-pub use schema::*;
+pub use spec::*;
 pub use traits::*;
 
 pub mod prelude {
-    pub use crate::json_ext::ValueExt;
+    // NOTE: only traits can be added here! Everything else should be scoped under the module
```

Comment:
good point there. The `ValueExt` trait is very specific to the way we manipulate JSON internally, it's not really necessary for router users

Reply:
Exactly! Unfortunately we need to keep it public a bit longer because the integration test uses it. I can add a `#[doc(hidden)]` :thinking: ...

Reply:
Do you mind doing it in your PR? If I did it here I would have had conflicts with your branch. 

You can just add `#[doc(hidden)]` just above the trait declaration.

Reply:
done in db4a518

Reply:
Thanks!